account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/AuthenticateHandler.java

```diff
@@ -72,12 +72,32 @@ public APIGatewayProxyResponseEvent authenticateRequestHandler(
                     objectMapper.readValue(input.getBody(), AuthenticateRequest.class);
             boolean userHasAccount = authenticationService.userExists(loginRequest.getEmail());
             if (!userHasAccount) {
+                auditService.submitAuditEvent(
+                        AccountManagementAuditableEvent.ACCOUNT_MANAGEMENT_AUTHENTICATE_FAILURE,
+                        AuditService.UNKNOWN,
+                        sessionId,
+                        AuditService.UNKNOWN,
+                        AuditService.UNKNOWN,
+                        loginRequest.getEmail(),
+                        IpAddressHelper.extractIpAddress(input),
+                        AuditService.UNKNOWN,
+                        AuditService.UNKNOWN);
```

Comment:
We seem to be extracting persistent session ID for `ACCOUNT_MANAGEMENT_AUTHENTICATE` using:

`PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders())`

Why not also for these?